qcodes/station.py

```diff
@@ -313,6 +326,27 @@ def get_config_file_path(
         with open(path) as f:
             self.load_config(f)
 
+    def load_config_files(self,
+                          *filenames: List[Union[str, Path]]
+                          ) -> None:
+        """
+        Loads configuration from multiple YAML files after merging them
+        into one. If `filenames` are not specified the default file name from
+        the qcodes configuration will be used.
+
+        Loading of configuration will update the snapshot of the station and
+        make the instruments described in the config files available for
+        instantiation with the :meth:`load_instrument` method.
+
+        Additionally the shortcut methods ``load_<instrument_name>`` will be
+        updated.
+        """
+        if len(filenames) == 1 and filenames[0] is None:
+            self.load_config_file()
+        else:
+            with _merge_yamls(*filenames[0]) as yamls:
+                self.load_config(yamls)
```

Comment:
This still has the same issue as before. If there is only one config file this is passed through load_config_file which will expand the path to the config file but if there is more than one config file the path is passed directly to _merge_yaml without being expanded.

Reply:
Yes, `get_config_file_path` is getting the file path in load_config_file method. Fixing this for new method as well.